File: PizzaApp/Entities/Pizza.cs
﻿namespace PizzaApp.Entities;

public class Pizza
{
    public int Id { get; set; }
    public string Name { get; set; }
    public string Ingredients { get; set; }
    public int Cost { get; set; }
    public int Weight { get; set; }
    public Photo Photo { get; set; }
}

File: pizza-client/src/app/_models/pizza.ts
import {Photo} from "./photo";
import {Toping} from "./toping";

export interface Pizza {
  id: number
  name: string
  photoUrl : string
  cost : number
  weight : number
  ingredients : string
  state : State
  stateAsString : string
  topings : Toping[]
  maxCost : number
}

export enum State{
  "Pending",
  "In Progress",
  "Ready",
  "Canceled"
}
